src/encoded/static/components/experiment_series.js

```diff
@@ -16,7 +16,7 @@ import { SortTablePanel } from './sorttable';
 import Status, { getObjectStatuses, sessionToAccessLevel } from './status';
 
 
-// Only GRCh38 analysis will be used for QC reporting
+// Only analysis on selcted assembly will be used for QC reporting
```

Comment:
Since this will be quick, might as well correct “selcted” to “selected.”

Reply:
Sorry. That's stupid. Fixed